docs/Clearwater_stress_testing.md

```diff
@@ -12,82 +13,112 @@ package installed.
 
 ### What they do
 
-Clearwater SIP stress nodes run a standard SIPp script against your bono cluster.  The bono nodes translate this into traffic for sprout and this generates traffic on homestead and homer.  The nodes log their success/failure to `/var/log/clearwater-sip-stress` and also restart SIPp (after a 30s cool-down) if anything goes wrong.
+Clearwater SIP stress nodes provide a set of SIPp scripts to run against your Sprout cluster. There are two kinds of stress available:
 
-The SIPp stress starts as soon as the node comes up - this is by design, to avoid you having to manually start 50+ nodes generating traffic.  If you want to stop them from generating traffic, turn the whole node down.
+* Our recommended approach is to use scripts which emulate a P-CSCF and send traffic to Sprout, stress testing the IMS Core directly.  The nodes log their success/failure to `/var/log/clearwater-sip-stress` and also print a human-readable summary of the stress (with information about percentage of failed calls, average latency, and so on).
+* We also have some older scripts which emulate UEs and send traffic to Bono, stress testing the P-CSCF and the IMS Core together.  The nodes log their success/failure to `/var/log/clearwater-sipp`, but unlike the other scripts, do not print a human-readable summary of the stress.
 
-Each SIP stress node picks a single bono to generate traffic against.  This bono is chosen by matching the bono node's index against the SIP stress node's index.  As a result, it's important to always give SIP stress nodes an index.
+## Deploying a stress node
 
-### How they work
+Follow [this process](https://github.com/Metaswitch/crest/blob/dev/docs/Bulk-Provisioning%20Numbers.md) to bulk provision the number of subscribers you want. As a general guideline, we'd expect a small deployment (with one Sprout, Homestead and optionally a Ralf node, each with one full CPU core) to handle at least 30,000 subscribers.
```

Comment:
Worth mentioning you need to spin up your clearwater deployment first, and linking to the docs?